src/main/kotlin/com/terraformation/backend/tracking/db/MonitoringPlotStore.kt

```diff
@@ -0,0 +1,30 @@
+package com.terraformation.backend.tracking.db
+
+import com.terraformation.backend.db.asNonNullable
+import com.terraformation.backend.db.default_schema.OrganizationId
+import com.terraformation.backend.db.tracking.MonitoringPlotId
+import com.terraformation.backend.db.tracking.PlantingSiteId
+import com.terraformation.backend.db.tracking.tables.references.MONITORING_PLOTS
+import jakarta.inject.Named
+import org.jooq.DSLContext
+
+@Named
+class MonitoringPlotStore(private val dslContext: DSLContext) {
+  fun getOrganizationIdsFromPlots(plotIds: Collection<MonitoringPlotId>): Set<OrganizationId> =
```

Comment:
I couldn't think of a way to add permissions checks to these that weren't doing N queries, or just retrieving the exact same thing. Open to ideas if we want these protected